specs/IsSmartScreenRequired

```diff
@@ -0,0 +1,94 @@
+In this document we describe the new setting. We'd appreciate your feedback.
+
+
+# Description
+You can use CoreWebView2Settings.IsSmartScreen to control SmartScreen. SmartScreen is enabled or disabled per browser process, so all WebView2 applications sharing the same user data folder path also share SmartScreen being enabled or disabled.
+If CoreWebView2Setting.IsSmartScreenRequired is true for any CoreWebView2 in its associated CoreWebView2Environment, then SmartScreen is enabled. If CoreWebView2Setting.IsSmartScreenRequired is false for all CoreWebView2s in their CoreWebView2Environment, then SmartScreen is disabled.
+The default value for `IsSmartScreenRequired` is true if SmartScreen is enabled when the CoreWebView2 is created and false if SmartScreen is disabled when the CoreWebView2 is created. The value doesn't change if SmartScreen is enabled or disabled later.
+
+Changes to `IsSmartScreenRequired` take effect on the next navigation or download.
+
+
+# Examples
+```cpp
+wil::com_ptr<ICoreWebView2Settings> m_webViewSettings;
+void SettingsComponent::ToggleSmartScreenRequired()
+{
+    wil::com_ptr<ICoreWebView2Settings11> coreWebView2Settings11;
+    coreWebView2Settings11 =
+                m_webViewSettings.try_query<ICoreWebView2Settings11>();
+    if(coreWebView2Settings11) 
+    {
+        BOOL enabled;
+        CHECK_FAILURE(coreWebView2Settings11->get_IsSmartScreenRequired(&enabled));
+        CHECK_FAILURE(coreWebView2Settings11->put_IsSmartScreenRequired(enabled ? FALSE : TRUE));
+    }
+}
+```
+
+```c#
+void ToggleSmartScreenRequired()
+{
+    var settings = webView2Control.CoreWebView2.Settings;
+    settings.IsSmartScreenRequired = !settings.IsSmartScreenRequired;
+}
+```
+
+# Remarks
+
+# API Notes
+
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+
+## Win32 C++
+```cpp
+[uuid(d667d3a7-c1b7-479f-8833-db7547df6687), object, pointer_default(unique)]
+interface ICoreWebView2Settings11 : ICoreWebView2Settings10 {
+  /// SmartScreen helps you identify reported phishing and malware websites  
+  /// and also helps you make informed decisions about downloads. 
+  /// `IsSmartScreenRequired`  is used to control whether SmartScreen enabled or not. 
+  /// SmartScreen is enabled or disabled for all CoreWebView2s in a CoreWebView2Environment. 
+  /// If CoreWebView2Setting.IsSmartScreenRequired is true for any CoreWebView2 in its associated 
+  /// CoreWebView2Environment, then SmartScreen is enabled. If CoreWebView2Setting.IsSmartScreenRequired 
+  /// is false for all CoreWebView2s in the associated CoreWebView2Environment, then SmartScreen is disabled.
+  /// When it is changed, the change will be applied to all WebViews using the 
+  /// same user data folder. 
```

Comment:
Somewhat confusing here that up until now, we talked about using the CoreWebView2Environment as the thing that we use to group CoreWebView2s, and then suddenly we switch and say "Tricked you. The thing that establishes grouping is the user data folder, not the CoreWebView2Environment."

So which is it?

Reply:
Please fix. Its mostly interchangeable. Probably CoreWebView2Environment makes more sense consistently.